politeiawww/cmd/piwww/testrun.go

```diff
@@ -45,9 +45,11 @@ func login(email, password string) error {
 	return lc.Execute(nil)
 }
 
-// newProposal returns a NewProposal object contains randomly generated
-// markdown text and a signature from the logged in user.
-func newProposal() (*v1.NewProposal, error) {
+// newProposal returns a NewProposal object contains randonly generated
+// markdown text and a signature from the logged in user, if given RFP
+// bool is true it creates an RFP.
+// If given linkto it creates a RFP submission.
+func newProposal(RFP bool, linkto string) (*v1.NewProposal, error) {
```

Comment:
Capitalizing the first letter of a variable in go means the variable is exported.  This is not an exported variable so it should be `rfp`, not `RFP`, to prevent any confusion.